models.py

```diff
@@ -892,19 +954,145 @@ class FeatureForm(forms.Form):
       widget=forms.Textarea(attrs={'cols': 50, 'placeholder': 'Summary description', 'maxlength': 500}),
       help_text='Complete sentences only. Provide a one sentence description followed by one or two lines explaining how this feature helps web developers.')
 
-  # owner = PlaceholderCharField(
-  #     required=False, placeholder='Owner(s) email',
-  #     help_text='Comma separated list of full email addresses (@chromium.org preferred).')
-
-  category = forms.ChoiceField(required=True,
+  category = forms.ChoiceField(required=True, help_text='Select the most specific category. If unsure, leave as "%s".' % FEATURE_CATEGORIES[MISC],
+      initial=MISC,
       choices=sorted(FEATURE_CATEGORIES.items(), key=lambda x: x[1]))
 
-  owner = forms.CharField(required=False, label='Owner(s) email',
+  intent_stage = forms.ChoiceField(required=True, label='Intent stage', help_text='Select the appropriate intent stage.',
+      initial=INTENT_IMPLEMENT,
+      choices=INTENT_STAGES.items())
+
+  owner = forms.CharField(initial=users.get_current_user().email, required=True, label='Contact emails',
       help_text='Comma separated list of full email addresses. Prefer @chromium.org.')
 
+  summary = forms.CharField(label='Feature summary', required=True, max_length=500,
+      widget=forms.Textarea(attrs={'cols': 50, 'placeholder': 'Summary description', 'maxlength': 500}))
+
+  motivation = forms.CharField(label='Motivation', required=True, max_length=500,
+      widget=forms.Textarea(attrs={'cols': 50, 'placeholder': 'Explain why the web needs this change. It may be useful to describe what web developers are forced to do without it. When possible, include links to back up your claims in the explainer.', 'maxlength': 500}))
+
+  explainer_links = forms.CharField(label='Explainer link(s)', required=True,
```

Comment:
Should this be required if, for some cases, only a blink-dev email will have gone out?